### PR TITLE
Adding vuid 04464 04462 04463, Validate GetImageSubresourceLayout aspect

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/buffer_validation.cpp [hunk 1]
```diff
@@ -6807,6 +6807,20 @@ bool CoreChecks::PreCallValidateGetImageSubresourceLayout(VkDevice device, VkIma
                 }
             }
         }
+        if (!FormatIsDepthAndStencil(img_format) && !FormatIsDepthOnly(img_format)) {
```
Comment (ncesario-lunarg) [inline layers/buffer_validation.cpp:6810]: I _think_ this is the case I was mistaking with the other change (sorry, too many reviews :)); I believe `FormatIsDepthAndStencil(img_format)` _is_ duplicated here.

**Review by ncesario-lunarg — APPROVED** [review]
One minor nit, but LGTM!

File: layers/buffer_validation.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/buffer_validation.cpp:6810]: I'm think both are needed again, first part of if checks for formats that include both depth and stencil, the second part of the if check for formats that are only depth.

**Review by ncesario-lunarg — APPROVED** [review]
Did you find out what the problem was with CI?

**Comment by ziga-lunarg** [comment]
> Did you find out what the problem was with CI?

Yes, I needed to call GetPhysicalDeviceFormatProperties, to check if format and tiling are supported, because InitNoLayout() will just change the tiling if it isn't.

**Pull request merged**